windows-start.bat

```diff
@@ -0,0 +1,15 @@
+@echo off
+REM Set the USDROOT environment variable to the relative path
+set USDROOT=.\build\usd
```

Comment:
That works but the build script should be installing in an `.\install` directory ? no ? Is it necessary to install then ?